elastic/kibana
# [Logs onboarding] Remove inspect button at the end of the flow

Closes https://github.com/elastic/kibana/issues/166256.

### Before
![image](https://github.com/elastic/kibana/assets/1313018/1f73cbb5-3acc-471d-9281-8c62d0ef0abb)

### After
<img width="1191" alt="image" src="https://github.com/elastic/kibana/assets/1313018/ab769f98-fe4f-407c-b21a-13b21c91f300">

### How to test
1. Go to [logs onboarding](https://yngrdyn-deploy-kiban-pr166423.kb.us-west2.gcp.elastic-cloud.com/app/observabilityOnboarding) landing page
2. Select `Stream log files` type of onboarding
3. Fill out the required fields
4. Go to next step
5. Verify that `Inspect` button is no longer next to `Explore logs` button
